plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.scss

```diff
@@ -34,9 +34,3 @@
 		border-bottom: 1px solid $gutenberg-gray-300;
 	}
 }
-
-@media screen and (min-width: $breakpoint-medium) {
-	.woocommerce-marketplace__tabs {
-		padding: 0 $content-spacing-large;
-	}
-}
```

Comment:
Designs don't have this padding. Everything is aligned this way:

![image](https://github.com/woocommerce/woocommerce/assets/10389957/a99dd88e-5d2f-4711-a395-a256f03c58e6)

Our page wasn't aligned because of this padding so I removed it. I did a similar thing in `discover.scss` as well.

Reply:
Thank you! The container already includes all the correct spacing, so this is a great find! Poli found a few more cases of stuff like this that we will handle in a tidy up PR.